cli/tests/e2e/test_ci.py

```diff
@@ -63,6 +63,34 @@
 FROZEN_ISOTIMESTAMP = "1970-01-01T00:00:00"
 
 
+# To ensure our tests are as accurate as possible, lets try to autodetect what GITHUB_ vars
+# the app code uses, so the tests can enforce the env is mocked appropriately.
```

Comment:
What is "the app code" in this context? Semgrep-app? the code in semgrep that interacts with the app?

Reply:
I'm using "app" here as the generic application, ie semgrep cli